### PR TITLE
startServerInstances and associatePublicIpWithServerInstance

### PR DESCRIPTION
**Pull request by stelcheck**
Adding the API to start servers..

### CODE REVIEW

**Comment by YuSungDuk** [comment]
@stelcheck Thank you for your PR!!. I'll review it.

File: sdk/associatePublicIpWithServerInstance_test.go [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+		AfterEach(func() {
+			gock.Off()
+		})
+		It("should Associate Public IP", func() {
+			reqParams := new(RequestAssociatePublicIP)
+			reqParams.PublicIPInstanceNo = "2501"
+			reqParams.ServerInstanceNo = "551236"
+
+			conn := NewConnection(accessKey, secretKey)
+			result, err := conn.AssociatePublicIPInstance(reqParams)
```
Comment (YuSungDuk) [inline sdk/associatePublicIpWithServerInstance_test.go]: maybe `AssociatePublicIP` ??

File: sdk/associatePublicIpWithServerInstance_test.go [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+		AfterEach(func() {
+			gock.Off()
+		})
+		It("should fail if The input parameter server instance number is invalid", func() {
+			reqParams := new(RequestAssociatePublicIP)
+			reqParams.PublicIPInstanceNo = "2501"
+			reqParams.ServerInstanceNo = "551236"
+
+			conn := NewConnection(accessKey, secretKey)
+			result, err := conn.CreatePublicIPInstance(reqParams)
```
Comment (YuSungDuk) [inline sdk/associatePublicIpWithServerInstance_test.go]: maybe `AssociatePublicIP` ??

**Comment by stelcheck** [comment]
Updated the PR. I haven't been able to run the tests locally yet, let me know if I need to change anything, however. I wrote this in haste, so it may be that I am doing something wrong :)

**Pull request merged**